rooms/rm_lvl_three/rm_lvl_three.yy

```diff
@@ -721,15 +721,19 @@
     {"name":"inst_4B4E9334","path":"rooms/rm_lvl_three/rm_lvl_three.yy",},
     {"name":"inst_3EFE04CA","path":"rooms/rm_lvl_three/rm_lvl_three.yy",},
     {"name":"inst_6D6E533B","path":"rooms/rm_lvl_three/rm_lvl_three.yy",},
+    {"name":"inst_3E904454","path":"rooms/rm_lvl_three/rm_lvl_three.yy",},
   ],
   "isDnd":false,
   "layers":[
-    {"$GMRInstanceLayer":"","%Name":"Player","depth":0,"effectEnabled":true,"effectType":null,"gridX":32,"gridY":32,"hierarchyFrozen":false,"inheritLayerDepth":false,"inheritLayerSettings":false,"inheritSubLayers":true,"inheritVisibility":true,"instances":[
+    {"$GMRInstanceLayer":"","%Name":"SettingsMenu","depth":0,"effectEnabled":true,"effectType":null,"gridX":32,"gridY":32,"hierarchyFrozen":false,"inheritLayerDepth":false,"inheritLayerSettings":false,"inheritSubLayers":true,"inheritVisibility":true,"instances":[
+        {"$GMRInstance":"v1","%Name":"inst_3E904454","colour":4294967295,"frozen":false,"hasCreationCode":false,"ignore":false,"imageIndex":0,"imageSpeed":1.0,"inheritCode":false,"inheritedItemId":null,"inheritItemSettings":false,"isDnd":false,"name":"inst_3E904454","objectId":{"name":"obj_settings_menu","path":"objects/obj_settings_menu/obj_settings_menu.yy",},"properties":[],"resourceType":"GMRInstance","resourceVersion":"2.0","rotation":0.0,"scaleX":1.0,"scaleY":1.0,"x":-32.0,"y":-32.0,},
```

Comment:
Does the settings object need to be added to each room, or can it just be added to the menu and exist as a persistent object throughout the rest of the game?

Reply:
From my understanding, if something is going to appear in a room, it needs to be in that room. Similar to the player object and camera and such, each room that they appear in gets their own instance. If we want to try and return to the "settings menu as a room" idea, I can see about figuring out how to change rooms without losing information from the previous room. I figured that setting it as a layer to only appear on top if the menu is opened was the simplest way, especially since we'd be doing a version of this already with a pause menu, so this could serve as a framework for that as well.

Reply:
my understanding was incorrect, settings menu is now a persistent object